admin-ui/plugins/auth-server/redux/sagas/authNLdapSaga.js

```diff
@@ -0,0 +1,118 @@
+import { call, put, takeLatest } from 'redux-saga/effects'
+import {
+  getLdapListSuccess,
+  getLdapListFailure,
+  deleteLdapSuccess,
+  deleteLdapFailure,
+} from '../features/authNLdapSlice'
+import { addLdapSuccess, addLdapFailure, editLdapSuccess } from '../features/authNLdapSlice'
+import { updateToast } from 'Redux/features/toastSlice'
+import { DatabaseLDAPConfigurationApi } from 'jans_config_api'
+import { initAudit } from '@/redux/sagas/SagaUtils'
+import { addAdditionalData } from '@/utils/TokenController'
+import { DELETION, FETCH, UPDATE } from '@/audit/UserActionType'
+import { postUserAction } from '@/redux/api/backend-api'
+import { LDAP } from '@/utils/ApiResources'
+
+function deleteLdapApi(configId) {
+  const api = new DatabaseLDAPConfigurationApi()
+  return new Promise((resolve, reject) => {
+    api.deleteConfigDatabaseLdapByName(configId, (error, data) => {
+      if (error) reject(error)
+      else resolve(data)
+    })
+  })
+}
+
+function* deleteLdapSaga({ payload }) {
+  const audit = yield* initAudit()
+  console.log('action in delete ldap saga', payload)
+  try {
+    addAdditionalData(audit, DELETION, LDAP, { message: payload?.userMessage })
+    yield call(deleteLdapApi, payload?.configId)
+    yield put(updateToast(true, 'success', 'LDAP deleted successfully'))
+    yield put(deleteLdapSuccess())
+    yield call(postUserAction, audit)
+    yield put({ type: 'authNLdap/getLdapList' })
+  } catch (e) {
+    yield put(updateToast(true, 'error', 'Error deleting LDAP'))
+    yield put(deleteLdapFailure())
+  }
+}
+function addLdapApi(payload) {
+  const api = new DatabaseLDAPConfigurationApi()
+  return new Promise((resolve, reject) => {
+    api.postConfigDatabaseLdap(payload, (error, data) => {
+      if (error) reject(error)
+      else resolve(data)
+    })
+  })
+}
+
+function* addLdapSaga(action) {
+  const audit = yield* initAudit()
+  try {
+    console.log('action in add ldap saga', action)
+
+    addAdditionalData(audit, FETCH, LDAP, { message: action?.payload?.action_message })
```

Comment:
Incorrect audit action type for add operation

The `addLdapSaga` uses `FETCH` as the action type when it should use a creation-related constant like `CREATE` or `ADD`.


```diff
-    addAdditionalData(audit, FETCH, LDAP, { message: action?.payload?.action_message })
+    addAdditionalData(audit, CREATE, LDAP, { message: action?.payload?.action_message })
```

Note: You'll need to import the appropriate constant (likely `CREATE` or `ADD`) from the audit action types.